Assembler.Base/EndFrameHandler.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using Assembler.Core;
+using Assembler.Core.Entities;
+using Assembler.Core.Enums;
+
+namespace Assembler.Base
+{
+    public class EndFrameHandler<TFrame, TMessage> : BaseFrameHandler<TFrame, TMessage>
```

Comment:
Please see the comments if the InitialFrameHandler and apply them here too

Reply:
f